hs/WebParsing/TimeTableParser.hs

```diff
@@ -14,17 +14,17 @@ import Database.JsonParser
 import WebParsing.HtmlTable
 import WebParsing.ParsingHelp
 import WebParsing.TimeConverter
-import Config (dbStr)
+import Config (databasePath)
 
 -- | used as an intermediate container while extracting lecture and tutorial information
 -- from the table. Is is later converted into lecture or tutorial records by examinig the
 -- first letter of the section.
 data CourseSlot =
-  CourseSlot {
-              slotSection :: T.Text,
-              slotTime_str :: T.Text,
-              slotInstructor :: T.Text
-              } deriving Show
+    CourseSlot {
+                slotSection :: T.Text,
```

Comment:
I'm not certain that the elements of containers should start on newlines. This style here is inconsistent with the rest of the codebase (specifically Database/Tables.hs).